plugin/rewrite/reverter_test.go

```diff
@@ -17,17 +16,14 @@ var tests = []struct {
 	answer   []dns.RR
 	to       string
 	toType   uint16
-	noRevert bool
 }{
-	{"core.dns.rocks", dns.TypeA, []dns.RR{test.A("dns.core.rocks.  5   IN  A  10.0.0.1")}, "core.dns.rocks", dns.TypeA, false},
-	{"core.dns.rocks", dns.TypeSRV, []dns.RR{test.SRV("dns.core.rocks.  5  IN  SRV 0 100 100 srv1.dns.core.rocks.")}, "core.dns.rocks", dns.TypeSRV, false},
-	{"core.dns.rocks", dns.TypeA, []dns.RR{test.A("core.dns.rocks.  5   IN  A  10.0.0.1")}, "dns.core.rocks.", dns.TypeA, true},
-	{"core.dns.rocks", dns.TypeSRV, []dns.RR{test.SRV("core.dns.rocks.  5  IN  SRV 0 100 100 srv1.dns.core.rocks.")}, "dns.core.rocks.", dns.TypeSRV, true},
-	{"core.dns.rocks", dns.TypeHINFO, []dns.RR{test.HINFO("core.dns.rocks.  5  HINFO INTEL-64 \"RHEL 7.4\"")}, "core.dns.rocks", dns.TypeHINFO, false},
```

Comment:
The tests here tested the `noRevert` flag which was deleted. Since it was deleted they serve no purpose now.

I wrote this explanation in my commit message here: https://github.com/coredns/coredns/pull/3273/commits/1c9eca6ff574cd489148af12848850433378ea49

To duplicate that message:


This also removes the tests for the `noRevert` behavior. The behavior
was never publicly exposed anyway, and was just used because the dns
recorder didn't capture requests robustly. After adding the
'mockDnsResponder' helper, that problem no longer exists.